alma/controllers/admin/AdminAlmaCategories.php

```diff
@@ -328,6 +324,23 @@ public function renderView()
         return $this->renderList();
     }
 
+    /**
+     * AdminController::renderList() override.
+     *
+     * @see AdminController::renderList()
+     */
+    public function renderList()
+    {
+        $this->addRowAction('voidaction');
```

Comment:
write `voidAction` instead of `voidaction`  (if possible)

Reply:
I fix it :) thanks